server/download.go

```diff
@@ -446,17 +447,17 @@ func watchDelta(ctx context.Context, g *LimitGroup, c *atomic.Int64, limit int64
 				continue
 			} else if maxDelta > 0 {
 				x := delta / maxDelta
-				if x < 1.2 {
+				if x < 1 {
```

Comment:
While better, I seem to get stuck in loops (perhaps from a max download speed per part from r2) where it's consistently _just below_ 1

<img width="841" alt="Screenshot 2024-03-07 at 5 41 31 PM" src="https://github.com/ollama/ollama/assets/251292/03019d2e-53e8-4e65-bd34-e3b22b41f1ea">